db/migrate/20220311005632_create_employees.rb

```diff
@@ -0,0 +1,14 @@
+class CreateEmployees < ActiveRecord::Migration[7.0]
+  def change
+    create_table :employees do |t|
+      t.string :name
+      t.string :email
+      t.string :position
+      t.integer :private_number
+      t.references :branch, null: false, foreign_key: true
+
```

Comment:
Since if you implement a session management gem such as [Devise](https://github.com/heartcombo/devise) or [Clearance](https://github.com/thoughtbot/clearance), the gem will automatically generate a user table with the email field, which would generate inconsistency with the email data that you have already generated in your tables.